test/nodetool/test_status.py

```diff
@@ -462,11 +462,8 @@ def test_status_keyspace_multi_dc(request, nodetool, uses_tablets, table):
     _do_test_status(request, nodetool, status_target, nodes)
 
 
-def test_status_keyspace_joining_node(request, nodetool):
-    """ Joining nodes do not have some attributes available yet:
-    * load
-    * host_id
-    """
```

Comment:
Explain why you remove `test_status_keyspace_joining_node` in the commit message -- it's not obvious. I see the explanation here: https://github.com/scylladb/scylladb/pull/20909#discussion_r1808428713. The rest of this commit message is good.

Also, the diff of `test_status.py` unfortunately doesn't look like it should. You removed one test and and added another. But instead of one red "-" block and one green "+" block, there is a hard to decipher mess. Maybe a simple trick like changing host ID of the first node in `test_status_with_zero_token_nodes` (so it differs from the node in `test_status_resolve`) would fix it. You can try. I don't think fixing it is necessary, though. It's not like you did something wrong.

Reply:
Done, added explanation!

>Maybe a simple trick like changing host ID of the first node in test_status_with_zero_token_nodes (so it differs from the node in test_status_resolve) would fix it

Had tried a few of these tricks, what happens is that the diff remains as it is right now, and just the new host_id string is green and the older one is made red, hence further increasing the chaos!